src/uct/tcp/tcp_iface.c

```diff
@@ -81,10 +81,29 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
                                 ucs_offsetof(uct_tcp_iface_config_t, rx_mpool), ""),
 
   {"PORT_RANGE", "0",
-   "Generate a random TCP port number from that range. A value of zero means\n "
+   "Generate a random TCP port number from that range. A value of zero means\n"
    "let the operating system select the port number.",
    ucs_offsetof(uct_tcp_iface_config_t, port_range), UCS_CONFIG_TYPE_RANGE_SPEC},
 
+#if UCT_TCP_EP_KEEPALIVE
+  {"KEEPIDLE", "10s",
+   "The time the connection needs to remain idle before TCP starts sending\n"
+   "keepalive probes.",
+   ucs_offsetof(uct_tcp_iface_config_t, keepalive.idle),
+                UCS_CONFIG_TYPE_TIME_UNITS},
+
+  {"KEEPCNT", "3",
+   "The maximum number of keepalive probes TCP should send before\n"
```

Comment:
no need \n

Reply:
tried to align line length with other lines.
removed ```\n```